camera_control/camera_control/camera/realsense.py

```diff
@@ -0,0 +1,97 @@
+import pyrealsense2 as rs
+import numpy as np
+
+
+class RealSense:
+    def __init__(self, logger, rgb_size, depth_size):
+        self.logger = logger
+        self.rgb_size = rgb_size
+        self.depth_size = depth_size
+
+    def initialize(self):
+        # Setup code based on https://github.com/IntelRealSense/librealsense/blob/master/wrappers/python/examples/align-depth2color.py
+        self.pipeline = rs.pipeline()
+        self.config = rs.config()
+        self.config.enable_stream(
+            rs.stream.depth, self.depth_size[0], self.depth_size[1], rs.format.z16, 30
+        )
+        self.config.enable_stream(
+            rs.stream.color, self.rgb_size[0], self.rgb_size[1], rs.format.bgr8, 30
```

Comment:
just curious - why BGR?

Reply:
opencv uses BGR as there image format, so I choose to use that in the beginning. As we are moving to using primarily models that expect RGB I can change the format to rgb8

Reply:
I think we should work with RGB wherever possible, and switch to BGR when we need to (openCV seems to be the only lib that uses it)